src/app/components/evolution/evolution.controller.js

```diff
@@ -13,13 +13,22 @@ angular.module('optimizer.evolution.controller', [])
         }, 0);
       };
 
-      $scope.getEvolutions = function (candy) {
+      $scope.getEvolutions = function (candy, transfer=false) {
```

Comment:
This may cause problems in browsers that don't have full ES2015 support. I suggest removing `=false` from this signature. (By default a parameter is `undefined` which will evaluate as `false` anyway, so this isn't really necessary).